unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/IAvatarSlotsView.cs

```diff
@@ -7,7 +7,7 @@ namespace DCL.Backpack
 {
     public interface IAvatarSlotsView
     {
-        event Action<string, bool> OnToggleAvatarSlot;
+        event Action<string, bool, bool> OnToggleAvatarSlot;
```

Comment:
you could declare a `delegate` so the parameters are more descriptive

Reply:
Done! https://github.com/decentraland/unity-renderer/pull/5028/commits/a4201f5d7425bcbe814c43ca65b310be0d6a007b